README.md

```diff
@@ -1 +1,138 @@
-# rails_capston
\ No newline at end of file
+<a name="readme-top"></a>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📰 Budget App <a name="about-project"></a>
+
+> Mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 📹 video presentation <a name="video-presentation"></a>
+
+- <a href="https://www.loom.com/share/eafb466b618942838e34989092fbe3eb?sid=41fb83d9-66f1-427b-8f3c-854a897d0afc">Video presentation link</a>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- <a href="https://www.ruby-lang.org/es/">ROR</a>
+- <a href="https://www.postgresql.org/">Postgresql</a>
+
+### Key Features <a name="key-features"></a>
+
+- User can create an account
+- User can login
+- User can create a transaction
+- user can create a category
+
+## Live Demo <a name="live-demo"></a>
+
+  - <a href="https://hasnatali1947.github.io/rails_capston/">Live Link</a>
```

Comment:
- [ ] your live demo link is leading to github page, kindly deploy your app and paste the link here

Reply:
"I've made multiple attempts to fix the live demo during deployment, but unfortunately, the issue persists. Could you kindly consider making this requirement optional? I'd greatly appreciate it. 🙏"

![image](https://github.com/hasnatali1947/rails_capston/assets/119807955/62b57383-dd38-47b7-95de-9b19365c081e)